src/Symfony/Component/HttpKernel/DataCollector/TimeDataCollector.php

```diff
@@ -51,7 +51,7 @@ public function collect(Request $request, Response $response, \Throwable $except
 
     public function reset()
     {
-        $this->data = [];
+        $this->data = ['events' => [], 'stopwatch_installed' => false, 'start_time' => 0];
```

Comment:
we should also do this in the constructor then

Reply:
fixed